draft-ietf-tsvwg-dtls-over-sctp-bis.md

```diff
@@ -1012,47 +1036,58 @@ terminated and the associated keying material discarded.
+   TLS-Exporter("EXPORTER-DTLS-OVER-SCTP-CLIENT-WRITE", , 64)
+   TLS-Exporter("EXPORTER-DTLS-OVER-SCTP-SERVER-WRITE", , 64)
    ~~~~~~~~~~~
 
-   When a subsequent DTLS connection is setup, a new 64-byte endpoint pair shared
-   secret is derived using the TLS-Exporter as defined above. The Shared Key
-   Identifiers form a sequence. If the previous endpoint pair shared secret used
-   Shared Key Identifier n, the new one MUST use Shared Key Identifier
-   n+1, unless n = 65535, in which case the new Shared Key Identifier
-   is 1.
+   When a subsequent DTLS connection is setup, two new 64-byte endpoint pair shared
+   secrets are derived using the TLS-Exporter as defined above. The Shared Key
+   Identifiers form a sequence. If the previous endpoint pair shared secrets used
+   Shared Key Identifiers 2n and 2n+1, the new ones MUST use Shared Key Identifier
+   2n+2 and 2n+3, unless 2n = 65534, in which case the new Shared Key Identifiers
+   are 2 and 3.
 
 ### DTLS 1.2 Considerations
 
-   Whenever a new DTLS connection is established, a 64-byte endpoint pair shared secret
-   is derived using the TLS-Exporter described in {{RFC5705}}.
+   Whenever a new DTLS connection is established, two 64-byte endpoint pair shared secrets
+   are derived using the TLS-Exporter described in {{RFC5705}}.
 
    After sending the DTLS Finished message
    for the initial DTLS connection, the active SCTP-AUTH key MUST be
-   switched from key identifier 0 to key identifier 1. Once the
+   switched from key identifier 0 to key identifiers 2 and 3. Once the
    initial Finished message from the peer has been processed by DTLS,
    the SCTP-AUTH Shared Key Identifier zero MUST NOT be used.
 
-   When the endpoint has both
-   sent and received a close_notify on the old DTLS connection then
-   the endpoint SHALL remove the SCTP-AUTH endpoint pair shared
-   secret derived from the old DTLS connection.
+   When the endpoint has sent or received a close_notify on the old DTLS
+   connection then the endpoint SHALL remove the two SCTP-AUTH endpoint
+   pair shared secrets derived from the old DTLS connection.
 
 ### DTLS 1.3 Considerations
 
-   Whenever a new exporter_secret can be computed, a 64-byte endpoint pair shared secret
+   Whenever a new exporter_secret can be computed, two 64-byte endpoint pair shared secrets
    is derived using the TLS-Exporter described in Section 7.5 of {{RFC8446}}.
 
-   TBD.... Finished and close_notify works differently in DTLS 1.3
+   After sending or receiving the DTLS server Finished message
+   for the initial DTLS connection, the active SCTP-AUTH key MUST be
+   switched from key identifier 0 to key identifiers 2 and 3 and
+   the SCTP-AUTH Shared Key Identifier zero MUST NOT be used.
+
+   When the endpoint has sent or received a close_notify in one direction
+   on the old DTLS connection then the endpoint SHALL remove the SCTP-AUTH endpoint
+   pair shared secret associated with that direction in the old DTLS connection.
```

Comment:
Should the requirement to retire the key before the TSN has wrapped once from the first TSN that contained a usage of this write SCTP-AUTH key be added here?